src/lib/util/Games/Slotmachine.ts

```diff
@@ -110,30 +102,40 @@ export class Slotmachine {
 		this.calculate(rolls);
 
 		const money = settings.get(UserSettings.Money);
-		const amount = this.winnings === 0 ? money - this.amount : money + (this.winnings * this.boost);
-		if (amount < 0) throw 'You cannot have negative money.';
+		const amount = this.winnings === 0 ? money - this.amount : money - this.amount + (this.winnings * this.boost);
+		if (amount < 0) throw this.message.language.tget('GAMES_CANNOT_HAVE_NEGATIVE_MONEY');
 		await settings.update(UserSettings.Money, amount);
-		return this.render(rolls);
+		return {
+			attachment: await this.render(rolls),
+			amount
+		};
```

Comment:
I'd prefer if this was an array that could be destructured but I couldn't figure out the typing on the command side of things where the destructuring happens. If someone could help here that be great.

Reply:
so returned as a tuple